inference-engine/tests/functional/inference_engine/serialization/single_layer/space_to_depth.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/space_to_depth.hpp"
+
+#include <ngraph/opsets/opset3.hpp>
+
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace ngraph::opset3;
+
+namespace {
+TEST_P(SpaceToDepthLayerTest, Serialize) {
```

Comment:
Could you add SLT?

Reply:
More SLT than that: 
- https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/single_layer_tests/space_to_depth.cpp
- https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/space_to_depth.cpp

Reply:
Don't we need to extend these tests?

Reply:
IMHO we got decent amount of SLT for SpaceToDetph.